test/utils-tests.jl

```diff
@@ -544,6 +544,20 @@ end
     [-0.37388888888888894, -1.0, -3.0]
 @test ten.([12.0, -0.0], [24, 30]) == ten.([" 12::24", " -0:30: "]) == [12.4, -0.5]
 
+#Test tic_once
+let
```

Comment:
Please, wrap all the tests for the functions with [`@testset`](http://docs.julialang.org/en/latest/stdlib/test.html#Base.Test.@testset), as I did [here](https://github.com/JuliaAstro/AstroLib.jl/commit/6644263e72a256271cd1a941a5ceb42d6b0fe024) for `ordinal` function.  This also avoids you using the `let` block.

Again, I didn't it for the other tests because this macro was introduced in Julia 0.5, but the last released version of AstroLib.jl still supports Julia 0.4.  I don't want you to wast your time using `@testset` for all other functions, I think I'll be able to do this next moth, but please start using it for the new tests ;-)